apps/ios/Shared/Views/Chat/ChatView.swift

```diff
@@ -502,6 +504,40 @@ struct ChatView: View {
     }
 }
 
+@ViewBuilder func toggleNtfsButton(_ chat: Chat) -> some View {
+    Button {
+        toggleNotifications(chat, enableNtfs: !chat.chatInfo.ntfsEnabled)
+    } label: {
+        if chat.chatInfo.ntfsEnabled {
+            Label("Mute", systemImage: "speaker.slash")
+        } else {
+            Label("Unmute", systemImage: "speaker")
```

Comment:
```suggestion
            Label("Unmute", systemImage: "speaker.wave.2")
```
or `speaker.wave.1` / `speaker.wave.3`?